extensions/amp-user-notification/0.1/amp-user-notification.js

```diff
@@ -232,12 +233,26 @@ export class AmpUserNotification extends AMP.BaseElement {
   }
 
   /**
-   * Get async cid service.
+   * Opts the user out of cid issuance and dismisses the notification.
+   * @private
+   */
+  optoutOfCid_() {
+    return this.getCidService_()
+        .then(cid => cid.optOutOfCid())
+        .then(this.dismiss.bind(this))
+        .catch(reason => {
```

Comment:
Nit: move this into the first then (nested), so that optout always dismisses regardless of error:

```js
cidService()
  .then(cid => cid.optOutOfCid().catch(reason => {}))
  .then(this.dismiss)
```